dali/pipeline/operator/arg_helper.h

```diff
@@ -105,6 +105,46 @@ class ArgValue {
     return has_arg_input_;
   }
 
+  /**
```

Comment:
Another variant of Acquire with an expected TensorListShape. We have one with an expected TensorShape (to be applied to each sample)